airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/extensions/LoggingInvocationInterceptor.kt

```diff
@@ -66,12 +66,19 @@ class LoggingInvocationInterceptor : InvocationInterceptor {
                     "instance creation for %s".formatted(invocationContext!!.targetClass)
             } else if (methodMatcher.matches()) {
                 val interceptedEvent = methodMatcher.group(1)
+                val methodRealClassName = invocationContext!!.executable!!.declaringClass.simpleName
+                val methodName = invocationContext.executable!!.name
+                val targetClassName = invocationContext!!.targetClass.simpleName
+                val methodDisplayName =
```

Comment:
what does this if/else handle? is it just for test methods declared in the base test class vs the actual test class?

Reply:
yeah, it's only used for logging, so we know where the actual method is, in addition to which test. So if you're executing `RealTest.testMethod()`, but `test Method` is implemented in `AbsractTest`, it'll print out 'RealTest.testMethod(AbstractTest)`